modules/install/pages/getting-started-docker.adoc

```diff
@@ -323,6 +325,130 @@ image::docker-single-machine-db123.png["The 'Servers' tab showing three nodes in
 
 Now that you have a multi-node Couchbase cluster running in containers across multiple physical hosts, you can move on to <<section_pfz_p1r_42b>>.
 
+[#section_deploy_multiple_clusters]
+== Deploy Multiple Clusters with Containers
+
+In this cluster deployment model, each cluster, running one or more nodes, is run in a separate container.
+All the containers run on a single physical host.
+When all containers run on a single physical host,
+it's important to remember that all containers will compete for the same resources.
+For this reason,
+it's not recommended to use this deployment model for use with applications that are sensitive to performance.
+
+The following procedure explains how to set up two clusters, each in a separate container,
+all running on one physical host.
+
+. Download and install Docker on the host computer.
++
+To set up Docker on the host computer,
+refer to Docker's https://www.docker.com/get-started[installation instructions^].
+
+. Install two instances of the official Couchbase Server container image.
++
+Make sure to run each of the following commands:
++
+[source,console]
+----
+$ docker run -d --name db1 -p 8091-8096:8091-8096 -p 11210-11211:11210-11211 couchbase
+----
++
+[source,console]
+----
+$ docker run -d --name db2 -p 9091-9096:8091-8096 -p 12210-12211:11210-11211 couchbase
```

Comment:
You can't re-use `12210-12211:11210-11211`, need to find a new set as you did with `8091`.